course_info/views.py

```diff
@@ -180,8 +159,8 @@ def editor(request):
         # "lis" appears to be a deliberate misspelling
         course_instance_id = request.POST.get('lis_course_offering_sourcedid')
 
-    # The values we will want to display
-    keys = [
+    # The values we will want to display, in the order we wish them to appear
+    field_names = [
```

Comment:
Yes! Good suggestion @rascalking. I need to get better acquainted with the [collections](https://docs.python.org/2/library/collections.html) library...